config/toml.go

```diff
@@ -372,6 +361,18 @@ func ResetTestRootWithChainID(testName string, chainID string) *Config {
 	return config
 }
 
+func mustCreateTestRootDir(testName, chainID string) string {
+	homeDir := os.ExpandEnv("$HOME/.tendermint_test")
+	if err := os.MkdirAll(homeDir, 0777); err != nil {
+		cmn.PanicSanity(err.Error())
+	}
+	rootDir, err := ioutil.TempDir(homeDir, fmt.Sprintf("%s-%s_", chainID, testName))
```

Comment:
Won't this clutter the home directory with test-directories (per chain-id), which are not cleaned up afterwards?Maybe add a comment which mentions that the caller is responsible of cleaning up the dirs after they are not used anymore.

Reply:
I deliberately avoided calling `defer os.RemoveAll(rootDir)` 'cause I had noticed that the latest test directory was always left on the system. On the other hand you're right, chances are that after multiple runs the user's home will be filled with test data. Happy to add a defer os.RemoveAll() :+1:

Reply:
Would be great to get rid of these directories https://github.com/tendermint/tendermint/issues/1034. But this will require a significant amount of time. Not sure it's worth doing right now unless you don't have anything else to do.

Reply:
done - please test

Reply:
wow. that was fast 🐇 ⚡️